lib/scope.js

```diff
@@ -1331,18 +1403,36 @@ module.exports = {
       }  // For each field
 
       await scope.afterStep( scope );
-    },  // Ends BirthDate()
+    },  // Ends scope.setCustomDatatype.al_date()
 
+    BirthDate: async function ( scope, { handle, set_to }) {
+      /** Given a date of the format 'mm/dd/yyyy' and the handle of the
+      * custom field container, sets the ALToolbox custom birthdate
+      * field values. */
+
+      await scope.setCustomDatatype.al_date( scope, { handle, set_to })
+    },  // Ends scope.setCustomDatatype.BirthDate()
+
+    ThreePartsDate: async function ( scope, { handle, set_to }) {
+      /** Given a date of the format 'mm/dd/yyyy' and the handle of the
+      * custom field container, sets the ALToolbox custom ThreePartsDate
+      * field values. */
+      await scope.setCustomDatatype.al_date( scope, { handle, set_to })
+    },  // Ends scope.setCustomDatatype.ThreePartsDate()
 
   },  // ends setCustomDatatype {}
 
-  setField: {
+  enter_answer: {
     button: async function ( scope, { handle, set_to }) { await scope.tapElement( scope, handle ); },
     textarea: async function ( scope, { handle, set_to }) {
       await scope.setText( scope, { handle, set_to });
       await scope.afterStep(scope);  // No showifs for this one?
     },
     select: async function ( scope, { handle, set_to }) {
+      // Note: I don't remember what's really going on in here.
+      // I will note that values of `select` elements are strings,
+      // not variables, so we don't need to check for variable names.
```

Comment:
Unless it's an object dropdown, which will have variable names as the option values.

Reply:
Variable names encoded in base64? I'll have to check on Slack. If so, I think we may need a new test to see if we need to sometimes encode `select` values.

Reply:
Yep! After making this comment, I actually ran into an bug that shows we don't handle this correctly, PR should be incoming.